scripts/drop_issuer_nf_tokens.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
+#Drops issuer_nf_tokens table and set latest ledger index to the one prior to XLS-20 amendment
```

Comment:
doesn't explain why it's setting the latest ledger index and why it's dropping the issuer_nf_tokens table. as a result this comment is not useful